pkg/kubectl/stop.go

```diff
@@ -509,3 +507,7 @@ func (reaper *ServiceReaper) Stop(namespace, name string, timeout time.Duration,
 	}
 	return services.Delete(name)
 }
+
+func getNonOrphanDeleteOptions() *api.DeleteOptions {
+	return &api.DeleteOptions{OrphanDependents: new(bool)}
```

Comment:
Yep agreed about the documentation, but readability wise 
`return &api.DeleteOptions{OrphanDependents: false}` vs `return &api.DeleteOptions{OrphanDependents: new(bool)}`, i would always pick the former :-). I agree the function name should be helpful as well.